salt/modules/grains.py

```diff
@@ -329,24 +332,38 @@ def append(key, val, convert=False, delimiter=DEFAULT_TARGET_DELIM):
     return setval(key, grains)
 
 
-def remove(key, val):
+def remove(key, val, delimiter=DEFAULT_TARGET_DELIM):
     '''
     .. versionadded:: 0.17.0
 
     Remove a value from a list in the grains config file
 
+    :param delimiter: The key can be a nested dict key. Use this parameter to
+        specify the delimiter you use.
+        You can now append values to a list in nested dictionnary grains. If the
+        list doesn't exist at this level, it will be created.
+        .. versionadded:: FIXME
```

Comment:
I didn't understand when to use version numbers or names. Now, I guess I need to put the name of the next release, and it will be changed to the version number just before the release. Right?